backend/main.go

```diff
@@ -106,6 +134,52 @@ func Run(cmd *cobra.Command, args []string) error {
 	handler := slog.NewJSONHandler(os.Stdout, nil)
 	logger := slog.New(handler)
 
+	// Try to massage unstructured Kubernetes logs into
+	// something that fits better with structured logs.
+	klog.SetLoggerWithOptions(
```

Comment:
`klog/v2` allows you to use some other logging backend, why not just put `slog` in there as a `logr.Logger`?

Reply:
Looks like [logr.FromSlogHandler](https://pkg.go.dev/github.com/go-logr/logr#FromSlogHandler) might work.

Reply:
Yep, it worked.  Updated 1st commit.